custom_components/bhyve/sensor.py

```diff
@@ -151,6 +147,7 @@ def __init__(self, hass, bhyve, device, zone):
             device,
             name,
             "history",
+            SensorDeviceClass.TIMESTAMP,
```

Comment:
It seems like this is the actual change in behaviour - is this correct?

(aside from the same change in the python script)